doc/operator-capabilities.md

```diff
@@ -149,32 +167,40 @@ The Operator offers one or more of the following lifecycle management features:
 
 2. Does your Operator expose Operand alerts?
 
-3. Does your Operator watch the Operand to create alerts?
+4. Do you have SOPs for each alert?
 
-4. Does your Operator emit custom Kubernetes events?
+5. Does you operator create critical alerts when the service is down and warning alerts for all other alerts?
 
-5. Does your Operator expose Operand performance metrics?
+6. Does your Operator watch the Operand to create alerts?
+
+7. Does your Operator emit custom Kubernetes events?
+
+8. Does your Operator expose Operand performance metrics?
 
 ---
 
 ## Level 5 - Auto Pilot
 
-**Auto-scaling**
+The final piece of the puzzle in eliminating/significantly reducing  any remaining manual intervention in managing the operand. The operator should automatically pick up on load of the operand and auto scale based on thresholds. The operator should automatically fix an unhealthy operand. The operator should tune the operands performance, this could include changing the node the pods are running on or modifying operand configuration.
```

Comment:
```suggestion
The highes capability level aims to significantly reduce/eliminate any remaining manual intervention in managing the operand. The operator should configure the Operand to auto-scale as load picks up. The Operator should understand  the application-level performance indicators and determine when it's healthy and performing well. The operator should attempt automatically fix an unhealthy operand. The operator should tune the operands performance, this could include scheduling on another node the pods are running on or modifying operand configuration.
```